docs/assignment2.md

```diff
@@ -1 +1,409 @@
+| F2   | Movements  | The player must move around the virtual environment using *move* followed by a direction: *North, East, West, South*. These command will move the player to the next virtual area. |
+| F3   | Narrator   | The narrator is a way for the system to interact with the user: Let the  user know what is going on. The narrator will describe the *field of view* of the player at any given time when: Moving between rooms, the command info is received. We define *field of view* as what all objects and properties of the immediate area around the  user. That is, the area the user is currently in. For example, if the  player is in a room that is dark, there is only an entrace to the west,  and in this area, there lies a fridge, broom and coffee machine. The  game would narrate *"You are in a dark room. The the west you see an  opening to another area. In this room, you see a fridge, broom and  coffee machine"*.  <br/>There are also other commands available,  which will have a special naration. For example, when picking up and  object, the narrator will reply with a simple confirmation, simply  repeating the action. If the action fails, the system should reply  accordingly. <br/>The command *diagnose* will trigger the narrator to list the health of the player. For example, you are wounded on your  left hand, 80% health remaining<br/>The command *inventory* will list all of the contents, without any specific ordering, the player currently has in their posession. |
+| F10  | Rooms      | There is a specific number of rooms that the player can enter, each resembling a room at the VU. |
+
+### 
+
+### Used modeling tool
+
+We are using draw.io
+
+###
+
+## Class diagram
+
+Author(s): Eduardo Lira & Theresa Schantz
+
+This chapter contains the specification of the UML class diagram of your system, together with a textual description of all its elements.
+
+```
+Class Diagram of VuORK
+```
+
+> Figure representing the Class Diagram of VuORK (Diagram 1) **DESCRIPTIVE**
+
+![Class Diagram of VuORK](https://i.postimg.cc/DZX98JfB/Class-diagram-Class-diagram.png)
```

Comment:
Multiplicity of the associations are not conformed with standard uml notation

Reply:
Thank you again :). Just to be clear, what do you mean 'only one is enough'. Looking at the <<Item>> abstract class, do we need only one line to Area/Player?

Also, I do not understand what is wrong with the multiplicity. Looking at the slides, it seems (to me), that if fits into the provided UML guidelines. 

Also, What did you think of the structure of the diagram?

Reply:
The multiplicity of the associations should not be enclosed with "[]", and the multiplicity, when displayed should appear on both ends of the association. Abstract classes should have "{abstract}". When you make an association between classes you do not have to display it as an attribute also (see slide 35 from the second lecture).

Reply:
Ah yes the "[]" was my bad. No clue why I added them. However, looking at slide 38 of Lecture 2, there is multiplicity given only at 1 end? That is what I followedas a guide to my mutiplicity. I ommitted the other multiplicty when there was a composite aggregation, because that means that Class A would not exist without Class B. 
Is this implicit description not valid UML?

Reply:
I checked the slide 38 and it shows the multiplicity at all ends of the association, as a ternary association. It is still necessary to show the multiplicity even in a composite aggregation (see slide 47 of lecture 2)